sycl/doc/extensions/experimental/sycl_ext_intel_esimd/README.md

```diff
@@ -95,6 +115,8 @@ done via explicit APIs; e.g. `sycl::ext::intel::experimental::esimd::block_store
 - `sycl::sampler` and `sycl::stream` classes
 
 ##### Other restrictions:
-- Only Intel GPU device is supported
+
+- Only Intel GPU device is supported in default mode. In ESIMD
```

Comment:
@kbobrovs , 'default' mode? or 'native' mode?

Reply:
default == native, right?

Reply:
This is the first place where the word 'default' shows up in this document. I wanted to make word choice consistent while you suggest introducing notion of 'native' and 'emulation' mode.